### ISSUE TITLE
Why the mask of decoder ?

### ISSUE THREAD

**Issue by zceng** [xcmyz/FastSpeech#16]
```
class Decoder(nn.Module):
    """ Decoder """

    def forward(self, enc_seq, enc_pos, return_attns=False):
        # ....
        # -- Prepare masks
        slf_attn_mask = get_attn_key_pad_mask(seq_k=enc_pos, seq_q=enc_pos)
        non_pad_mask = get_non_pad_mask(enc_pos)
```
为什么使用 `enc_pos` 来计算 mask ，难道不应该是使用 `enc_seq` 吗？

**Comment by superhg2012** [comment]
@JackZiLong have you figured out this question?
